src/UI/worldScreen/TileSidebar.ts

```diff
@@ -88,10 +88,16 @@ export default class TileSidebar implements Page {
         if (project.costs.length == 0) {
             projectHTML.appendChild(UI.makePara("Cost: Free"));
         } else {
-            const cssClass = this.run.inventory.canAfford(project.costs) ? "project-requirement-met" : "project-requirement-unmet";
-            const costDescriptions = project.costs.map((cost: Cost) => `${cost.toString()}`);
-            const costsString = "Cost: " + costDescriptions.join(", ");
-            projectHTML.appendChild(UI.makePara(costsString, [cssClass]));
+            const costParas = project.costs.map((cost: Cost) => UI.makePara(cost.toString(),
+                [
+                    this.run.inventory.canAfford([cost])
```

Comment:
there is a potential bug here that can result if a tile project had 2 associated costs that use the same resource, which is unlikely but should be considered valid.

say that a project's costs were `[new Cost(Resource.Energy, 100), new Cost(Resource.Energy,100)]` and the player had 150 units of energy in their inventory. In this case, each call to `inventory.canAfford()` would return true, even though the player cannot afford the overall cost.

one way to work around this is by using a clone of the inventory, and deducting the cost from that clone after evaluating each cost. maybe take a look at the production screen code to see how this works.